src/mumble/TalkingUI.cpp

```diff
@@ -752,6 +752,13 @@ void TalkingUI::on_settingsChanged() {
 		}
 	}
 
+	// Update Background Color
+	for (auto& currentContainer : m_containers) {
+		for (auto &currentEntry : currentContainer->getEntires()) {
```

Comment:
This does not conform to our coding guidelines: https://github.com/mumble-voip/mumble/blob/master/CODING_GUIDELINES.md#use-of-auto

(`auto` makes the code harder to read because it's not clear as to what the type of these variables is)

Reply:
I chose to use the auto keyword here because it matches the style of the code around it, should I use syntax similar to the for loop on line 40? `For (std::stize_t i = 0; i < m_containers.size(); i++) {}`

Reply:
Oh yes indeed - I wasn't even aware that I used `auto` here :eyes: 
In that case, leave it as is.